zero_bin/rpc/src/main.rs

```diff
@@ -219,6 +219,7 @@ async fn main() -> anyhow::Result<()> {
     tracing_subscriber::Registry::default()
         .with(
             tracing_subscriber::fmt::layer()
+                .with_writer(std::io::stderr)
```

Comment:
nit: you probably wanna keep the comment explaining this is to distinguish it from the output payload

Reply:
Alright.